sklearn/metrics/classification.py

```diff
@@ -282,7 +282,9 @@ def cohen_kappa_score(y1, y2, labels=None, weights=None):
     assigned to any sample (the observed agreement ratio), and :math:`p_e` is
     the expected agreement when both annotators assign labels randomly.
     :math:`p_e` is estimated using a per-annotator empirical prior over the
-    class labels [2].
+    class labels [2]_.
+
+    Read more in the :ref:`User Guide <cohen_kappa_score>`.
```

Comment:
I assumed that adding this line would link to the User Guide page, but it didn't work. @jnothman, is there anything else that needs to be added here?